src/index.js

```diff
@@ -275,13 +310,18 @@ async function notifyMessage(title, columnID) {
   );
 
   parsedData.forEach((items) => {
-    dataToSend += `@${items.name}`;
-    dataToSend += '\n';
+    /** Skip person with no tasks */
+    if (!items.tasks.length) {
+      return;
+    }
```

Comment:
@talyguryn, Peter told me earlier to remove this, So we can identify a person without a task.

Reply:
hm. well then we can mark these people other way. maybe put them at the end of the message and mark as "no task" or any other way.

i did this because i see big message which has only 50% useful information. let's discuss a better design format.